### DISCUSSION TITLE
Frontend filtering a table based on a relationship to another table

### DISCUSSION THREAD

**Discussion by bmscmoreira** [Budibase/budibase#12043] [Help] [answered]
I have a "Movements" table with a column that is a "many-to-one" relationship to another table called "Places".
(When I add a row in "Movements" table I can pick one value from the "Places" table to display in the "Movements" table.

Is it possible to display the "Movements" table in the design view with a filter based on the "Places" table?

I can display the table:
<img width="949" alt="2023-10-11_10-43-58_PM" src="https://github.com/Budibase/budibase/assets/3017910/ae0e855c-3590-4600-ad35-2f7b597a4f3d">

And get the options field pull the right values:
<img width="936" alt="2023-10-11_10-44-17_PM" src="https://github.com/Budibase/budibase/assets/3017910/8f0aabdf-2639-454b-a933-ce04b67c0673">

But I can't tell the "Movements" table to filter based on the selected option.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @bmscmoreira,

Have you checked out the documentation [here](https://docs.budibase.com/docs/filter-by-relationships#tutorial-filter-on-selected-relationship-picker-values)?

**Reply by bmscmoreira** [reply]
Thanks. I did try relationship picker but had no success as well, but after reading the documentation (with more attention) I realized that I'm not using a relational database, but the internal one. So that must be the issue.
